### PR TITLE
Add caret to signify dropdown menu

### PR DESCRIPTION
**Pull request by stevebauman**
Sorry, don't know how to merge commits! :P

### CODE REVIEW

**Comment by crynobone** [comment]
Would be cleaner with a single if else condition.

**Comment by stevebauman** [comment]
Yea you're right, how's that?

File: resources/views/components/usernav.blade.php [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
+	    @if(is_null($user))
+		<a href="#user-menu" rel="user-menu">
+			<i class="icon-user"></i>
+			{{ trans('orchestra/foundation::title.login') }}
 		</a>
-		@unless(is_null($user))
+		@else
+		<a href="#user-menu" rel="user-menu" class="dropdown-toggle" data-toggle="dropdown">
+		    <i class="icon-user"></i>
+		    {{ ! is_null($user) ? $user->fullname : trans('orchestra/foundation::title.login') }}
```
Comment (crynobone) [inline resources/views/components/usernav.blade.php]: Can remove this condition. Just show the user fullname.

**Comment by stevebauman** [comment]
Whoops there we go, hows that now?

**Comment by crynobone** [comment]
Thanks

**Pull request merged**